src/classes/CON_Preferred_TEST.cls

```diff
@@ -37,92 +37,116 @@
+        System.assertEquals(true, preferredPhoneHandler[0].active__c);
+      
+        
+        if (preferredPhoneHandler != NULL && preferredPhoneHandler.size() > 0) {
+        
+            preferredPhoneHandler[0].active__c = false;
+            update preferredPhoneHandler[0];
+        
+        }
+        
+        Contact contact1 = new Contact(LastName = 'TestersonA', UniversityEmail__c = 'fake@test1.com', Preferred_Email__c = 'University', 
+            HomePhone = '555-432-4433', PreferredPhone__c = 'Home');
+        Contact contact2 = new Contact(LastName = 'TestersonB', WorkEmail__c = 'fake@test2.com', Preferred_Email__c = 'Work', 
+            WorkPhone__c = '555-432-4433', PreferredPhone__c = 'Work');
+        Contact contact3 = new Contact(LastName = 'TestersonC', AlternateEmail__c = 'fake@test3.com', Preferred_Email__c = 'Alternate', 
+            OtherPhone = '555-432-4433', PreferredPhone__c = 'Other');
+        Contact contact4 = new Contact(LastName = 'TestersonD', AlternateEmail__c = 'fake@test4.com',Preferred_Email__c = 'Alternate',
+            MobilePhone = '555-432-4433', PreferredPhone__c = 'Mobile');
+            
+        List<Contact> contacts = new Contact[]{contact1, contact2, contact3, contact4};
+        Test.startTest();
+        insert contacts;
+        Test.stopTest();
+        
+        contacts = [SELECT Email, LastName, Phone, UniversityEmail__c, WorkEmail__c, AlternateEmail__c, HomePhone, WorkPhone__c, OtherPhone, MobilePhone 
+                    FROM Contact where ID in :contacts];
+        
+        //ensure we can retrieve the contacts in a specific order/know which contact we are comparing
+        //can no longer user Name in filters if we want to be Shield compatible
+        Map<String, Contact> contactsByLastName = new Map<String, Contact>();
+        for(Contact c : contacts) {
+            contactsByLastName.put(c.LastName, c);
+        }
+        contact1 = contactsByLastName.get('TestersonA');
+        contact2 = contactsByLastName.get('TestersonB');
+        contact3 = contactsByLastName.get('TestersonC');
+        contact4 = contactsByLastName.get('TestersonD');
+        
+        System.assertEquals(contact1.UniversityEmail__c, contact1.Email);
+        System.assertEquals(contact2.WorkEmail__c, contact2.Email);
+        System.assertEquals(contact3.AlternateEmail__c, contact3.Email);
+        
+        System.assertEquals(contact1.HomePhone, contact1.Phone);
+        System.assertEquals(contact2.WorkPhone__c, contact2.Phone);
+        System.assertEquals(contact3.OtherPhone, contact3.Phone);
+        System.assertEquals(contact4.MobilePhone, contact4.Phone);
+    }
+    
+    @isTest
+    public static void preferredEmailPhoneUpdate() {
+    
+        Test.testInstall(new STG_InstallScript(), null);
+        List<trigger_handler__c> preferredPhoneHandler = [select id, name, active__c from trigger_handler__c WHERE class__c = 'CON_PreferredPhone_TDTM' AND Object__c = 'Contact'];
```

Comment:
more SOQL formatting stuff, lets try to keep the all caps format for SOQL commands "SELECT Id, blah FROM...WHERE...", etc